src/reanimated2/component/KeyboardAvoidingView.tsx

```diff
@@ -22,13 +22,14 @@ try {
   useHeaderHeight = () => 0;
 }
 
-type Props = ViewProps & KeyboardAvoidingViewProps;
-
 /**
  * View that moves out of the way when the keyboard appears by automatically
  * adjusting its height, position, or bottom padding.
  */
-const KeyboardAvoidingView = forwardRef<View, React.PropsWithChildren<Props>>(
+export const ReanimatedKeyboardAvoidingView = forwardRef<
```

Comment:
Why not just `KeyboardAvoidingView`?

Reply:
@tomekzaw tried to name it similarly to our other components like `AnimatedView` or `AnimatedText`, but if you say so, I will revert it

Reply:
Both `AnimatedView` and `AnimatedText` are higher-order components (HOC) created with `createAnimatedComponent` function. In this case, `KeyboardAvoidingView` is not an animated component so we can export it under the same name.